bin/brew

```diff
@@ -1,7 +1,10 @@
-#!/bin/sh
+#!/bin/bash
 
 chdir() {
-  cd "$@" >/dev/null
+  cd "$@" >/dev/null || {
+    echo "Error: failed to cd to " "$@" "!" >&2
```

Comment:
*Addendum:* Note also that `"Error: failed to cd to $*!"` feels a bit less awkward than `"Error: failed to cd to " "$@" "!"`. (And it gets rid of the superfluous white space.)